common/src/IO/Path.cpp

```diff
@@ -311,9 +312,9 @@ namespace TrenchBroom {
 
         bool Path::hasFilename(const std::string& filename, const bool caseSensitive) const {
             if (caseSensitive) {
-                return kdl::cs::is_equal(filename, this->filename());
+                return kdl::cs::str_is_equal(filename, this->filename());
```

Comment:
is there a reason to have `kdl::cs::str_is_equal` over just `==`?

Reply:
Not really, it's changed now.